### ISSUE TITLE
Access matches database externally

### ISSUE THREAD

**Issue by Skoshche** [elovation/elovation#118]
Hello, is there any way to access and modify the database externally? 

I would like to modify the dates for the matches that took place before implementing the system but cannot find anything allowing me to do so, as well as batch edit anything else. Thanks

**Comment by christoshrousis** [comment]
There isn't currently a way to import matches. It is running a postgresql instance, and you can theoretically connect to the database and attempt to insert records. But the problem is that the ELO rankings won't get calculated.

If you don't have a lot of records, I suggest trying to enter them manually is possible, or just use the system for only new matches moving forward?

I guess this would make a good feature for people to be able to bulk import matches that have historically occurred.

**Comment by Skoshche** [comment]
Yeah sadly that's about what I expected and where I was getting hung up on my own program for this same sort of thing
Only way I could think to solve it was having it possible to change dates but only to a date before the previous and after the last, keep the calculation order
